src/main/java/InvalidArgumentAelitaException.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Signals that the argument in invalid. <br>
+ * Thrown when a negative number is used as argument for setTotalTaskCount().
+ */
+public class InvalidArgumentAelitaException extends AelitaException {
+
+    /**
+     * Constructs a new instance of InvalidArgumentAelitaException.
+     */
+    public InvalidArgumentAelitaException() {
```

Comment:
You do not require a blank line after the method definition. Please check the oracle code conventions and fix it.